test/test_sparse_csr.py

```diff
@@ -955,6 +957,25 @@ def test_coo_csr_conversion(self, device, dtype):
 
             self.assertEqual(csr_sparse.to_dense(), dense)
 
+    @ops(_sparse_csr_ops)
```

Comment:
Let's have the filter only for the unary ops that support sparse csr input.

Reply:
Done. :)